src/stylesheets/components/_navigation.scss

```diff
@@ -146,38 +207,76 @@ $nav-link-arrow-icon-size: 2;
 
     &[aria-expanded="false"] {
       /* stylelint-disable-line selector-no-qualifying-type */
-      @include add-background-svg("usa-icons/add");
-      background-position: right 0 center;
-      background-size: units($nav-link-accordion-icon-size);
+      background-image: none;
+
+      span {
+        @include place-icon($add-icon, "after");
+
+        &::after {
+          position: absolute;
+          right: 0;
```

Comment:
# Header

In high contrast modes, the header would be missing chevrons, mobile open/close buttons, and the search button depending on theme color

_(before and after)_
![image](https://user-images.githubusercontent.com/25211650/160002288-92199d22-a325-4db0-8761-9a6c115e98e7.png)

_Mobile_
![image](https://user-images.githubusercontent.com/25211650/160002333-fc3bde48-e8e0-4d65-b011-99a0dfa5a1b4.png)